internal/config/types.go

```diff
@@ -20,11 +20,17 @@ type ControlServer struct {
 }
 
 // Metrics config.
+// Deprecated.
 type Metrics struct {
-	Enabled    bool   `yaml:"enabled" json:"enabled" env:"ENABLED"`
 	ClientType string `yaml:"client_type" json:"client_type" env:"CLIENT_TYPE" env-default:"noop"`
 }
 
+// PrometheusMetrics config.
+type PrometheusMetrics struct {
+	Enabled bool   `yaml:"enabled" json:"enabled" env:"ENABLED"`
```

Comment:
Let's actually rename this to just `Enable` to match that of the Postgres TLS config & the global TLS config... least we can get it to be consistent.

```suggestion
	Enable bool   `yaml:"enable" json:"enable" env:"ENABLE"`
```